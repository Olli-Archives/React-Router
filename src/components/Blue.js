import React from 'react';

export default function Blue(){
  return (
    <div style={{ width:'100px', height:'100px', backgroundColor:'blue' }}>

    </div>
  );
}
