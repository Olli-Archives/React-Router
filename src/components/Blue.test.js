import React from 'react';
import { shallow } from 'enzyme';
import Blue from './Blue';


describe('Blue component test', ()=>{
  it('renders stuff', ()=>{
    const wrapper = shallow(<Blue/>);
    expect(wrapper).toMatchSnapshot();
  });
})
;
