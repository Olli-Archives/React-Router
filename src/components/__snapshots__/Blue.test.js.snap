// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Blue component test renders stuff 1`] = `
<div
  style={
    Object {
      "backgroundColor": "blue",
      "height": "100px",
      "width": "100px",
    }
  }
/>
`;
